script.module.dateutil/addon.xml

```diff
@@ -1,22 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.dateutil"
-       name="python-dateutil"
-       version="2.8.1+matrix.1"
-       provider-name="Paul Ganssle, Gustavo Niemeyer">
+       name="dateutil"
+       version="2.8.2+matrix.1"
+       provider-name="Gustavo Niemeyer, Tomi Pieviläinen, Yaron de Leeuw, Razzeee">
```

Comment:
Have you patched the module or is it plain upstream? If it's plain upstream no need to define as an author